app/helpers/application_helper.rb

```diff
@@ -151,4 +151,15 @@ def time_ago_in_words_label(time)
     ago = time_ago_in_words(time)
     content_tag(:span, ago, title: time.strftime("%F %T %z"))
   end
+
+  def highlight_search_terms(text, search_terms)
```

Comment:
I'm not sure here, but maybe pass in the `Search` object? It seems odd to extract terms from `@search` but only pass them into this method. I'm not sure how it'll look after the refactor around `strip_operators` though.